src/renderer/tvgScene.cpp

```diff
@@ -70,7 +70,9 @@ Type Scene::type() const noexcept
 Result Scene::push(Paint* paint) noexcept
 {
     if (!paint) return Result::MemoryCorruption;
-    PP(paint)->ref();
+    auto pPaint = PP(paint);
+    pPaint->ref();
+    pPaint->renderFlag |= RenderUpdateFlag::Transform;
```

Comment:
I suggest updating the scene directly.
```
    PP(pImpl->scene)->renderFlag |= RenderUpdateFlag::All;
```

Reply:
It seems to me that in this case, each time I push paint to the scene it will update all paints in that scene. Are there any situations where such a broad update would be required?

Reply:
On second thought, you have a point. We can go with an individual update. Just I've also considered 2 cases more.

1. In case of canvas push.  (we can deal with this separately)
2. To the same scene push/clear repeatedly without any practical updates.

Reply:
1. With Canvas push the problem did not occur, probably due to the [update call with the force flag](https://github.com/thorvg/thorvg/blob/511495f6d82b20acbe4ac3983a2aae9a3786b76b/src/renderer/tvgCanvas.h#L70). 

2. Maybe a second argument in the push method to raise the update flag? Or a separate method to do this? Automating flag raising can be difficult because the scene may be transformed between clear and push calls.